Tests/NautilusTelemetryTests/TraceExporterTests.swift

```diff
@@ -205,7 +205,8 @@ final class TraceExporterTests: XCTestCase {
 		let traceId2 = Identifiers.generateTraceId()
 
 		let span1 = Span(name: "root", traceId: traceId1, parentId: nil)
-		let span2 = Span(name: "hello", traceId: traceId2, parentId: nil, linkedParent: span1)
+		let span2 = Span(name: "hello", traceId: traceId2, parentId: nil)
+		span2.addLink(span1, relationship: .parent)
```

Comment:
Why not always create a link from a child span to a parent span? I suspect there's a good reason but I was curious so I wanted to ask.